app/serializers/api/public/v1/serializable_course.rb

```diff
@@ -16,6 +16,12 @@ def enrichment_attribute(name, enrichment_name = name)
           data { @object.accrediting_provider }
         end
 
+        has_many :location_statuses do
+          data do
+            @object.site_statuses
+          end
+        end
+
```

Comment:
Documentation needs updating

ie 
![image](https://user-images.githubusercontent.com/470137/104161241-695bab00-53ea-11eb-98d0-940a8a1d81e2.png)

Reply:
You mean this?
https://github.com/DFE-Digital/teacher-training-api/pull/1718/files#diff-4427947212e294ec5b97c1345d508c970654f96036e1f9e7bf130ccb460843ebR48

Reply:
I think also this files needs to be changed

https://github.com/DFE-Digital/teacher-training-api/blob/master/swagger/public_v1/api_spec.json

Reply:
I am now using another end point to fetch locations - changes no longer required